test/metrics.bats

```diff
@@ -97,21 +97,13 @@ function teardown() {
 	CONTAINER_ENABLE_METRICS=true CONTAINER_METRICS_PORT=$PORT start_crio
 
 	jq '.linux.resources.memory_limit_in_bytes = 15728640
-        | .command = ["sh", "-c", "dd if=/dev/zero of=/dev/null bs=20M"]' \
+        | .command = ["sh", "-c", "sleep 1; dd if=/dev/zero of=/dev/null bs=20M"]' \
 		"$TESTDATA/container_config.json" > "$TESTDIR/config.json"
 	CTR_ID=$(crictl run "$TESTDIR/config.json" "$TESTDATA/sandbox_config.json")
 
-	# Wait for container to OOM
-	CNT=0
-	while [ $CNT -le 100 ]; do
-		CNT=$((CNT + 1))
-		OUTPUT=$(crictl inspect --output yaml "$CTR_ID")
-		if [[ "$OUTPUT" == *"OOMKilled"* ]]; then
-			break
-		fi
-		sleep 10
-	done
-	[[ "$OUTPUT" == *"OOMKilled"* ]]
+	# Wait for container to OOM.
+	EXPECTED_EXIT_STATUS=137 wait_until_exit "$CTR_ID"
+	crictl inspect "$CTR_ID" | jq -e '.status.reason == "OOMKilled"'
```

Comment:
can we also compress the `ctr.bats` `ctr oom` test into this one?

Reply:
@haircommander not sure what do you mean, I can't find any `ctr oom` test.

Ah! This was already done in commit a1c6ae7e4eb98ef0d511946bcf28e34a345a5876

Reply:
I mean, commit  e914f0e15751fe4903c01f64632ec2fb1f60cb41